vite.config.js

```diff
@@ -0,0 +1,19 @@
+import { resolve } from 'path'
+import { defineConfig } from 'vite'
+import { nodeResolve } from '@rollup/plugin-node-resolve'
+
+export default defineConfig({
+  plugins: [nodeResolve()],
+  server: {
+    watch: {
+      ignored: ['**/_opam']
+    }
+  },
+  build: {
+    rollupOptions: {
+      input: {
+        better_sandwiches: resolve(__dirname, 'src/better-sandwiches/index.html'),
```

Comment:
I don't understand why this is only needed for one chapter.

Reply:
This was left in from where I explain how to make a bundle so you can compare the size with and without `Printf`. But I think it's a bit confusing to leave it in, so I delete it in https://github.com/melange-re/melange-for-react-devs/pull/16/commits/5f3e93ed7834d8d28dd81f7c6e3c0ca57f44cbf7